python/proton/_wrapper.py

```diff
@@ -134,6 +150,12 @@ def __repr__(self):
                                         self.__class__.__name__,
                                         id(self), addressof(self._impl))
 
+# print(f"log _wrapper line 144 {pn_py2void(Wrapper)=} ")
+# PYCTX = int(pn_py2void(Wrapper))
+# print(f"log _wrapper line 144 : type {type(Wrapper)} \n wrapper {Wrapper}")
+handle = ffi.new_handle(Wrapper)
```

Comment:
@jiridanek I'm sure about this approach. Using ffi.new_handle() has return random unique value. This didn't raise any errors here